addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1289,6 +1292,9 @@ msgctxt "#297"
 msgid "Audio offset"
 msgstr ""
 
+#: addons/skin.confluence/720p/VideoOSD.xml
+#: skin.confluence/720p/VideoOSDBookmarks.xml ## used as <label>$LOCALIZE[298] / $LOCALIZE[295]</label>
```

Comment:
Context comments start with  #.  And give a description of what is used for and not a repetition of skin code